src/rules/program/target_end.py

```diff
@@ -0,0 +1,41 @@
+from time import strftime
+
+import jira
+
+from utils.jira import refresh, update
+
+today = strftime("%Y-%m-%d")
+
+
+def check_target_end_date(
+    issue: jira.resources.Issue, context: dict, dry_run: bool
+) -> None:
+    children = issue.raw["Context"]["Related Issues"]["Children"]
+
+    if not children:
+        return
```

Comment:
The target end date should be unset (which I think it would be if you didn't return).

Reply:
Done in 84ade80.